v5_MIGRATION_GUIDE.md

```diff
@@ -223,7 +238,7 @@ class MyActivity : FragmentActivity() {
     
     override fun onCreate(savedInstanceState: Bundle?) {
 +       // can initialize clients outside of onCreate if desired
```

Comment:
The `GooglePayLauncher` uses the android activity result API under the hood (registers for activity results from Google Pay SDK) so it actually _does_ need to be instantiated in `onCreate`(the client doesn't need to be though) - idk if calling out that the others dont is helpful or confuses things. What do you think?

Reply:
Ah I see. Yeah maybe calling out that the client doesn't need to be created in `onCreate` would cause more confusion. Were there merchants who needed to instantiate Client classes outside of 
onCreate?

Reply:
Yes one of the problems in v4 is that the clients need to be instantiated in `OnCreate` because they use that activity result API, but they also need authorization, which merchants often fetch async. So breaking them into two classes, one that needs auth and can be instantiated after `OnCreate`, and one that doesn't need auth but does need to be instantiated in `OnCreate` was the solution.

Reply:
Got it! What do you think about adding comments for both launcher and client creation for each integration? That could help remove confusion.

Reply:
Yes sounds good!